w3af/plugins/crawl/open_api.py

```diff
@@ -407,6 +415,13 @@ def get_options(self):
         o = opt_factory('discover_fuzzable_headers', self._discover_fuzzable_headers, d, BOOL, help=h)
         ol.add(o)
 
+        d = 'Path to a file with parameter values.'
+        h = ('This option sets a path to a YAML file which contains parameter values'
+             ' which should be used in testing API endpoints. If no parameter values are provided,'
+             ' the plugin tries to guess them.')
+        o = opt_factory('parameter_values_location', self._parameter_values_location, d, INPUT_FILE, help=h)
```

Comment:
`parameter_values_location` should be changed to `parameter_values_file` (that is how input files are usually named)

https://github.com/andresriancho/w3af/blob/356b14b975039706f4fd7f4f5db5b114cd75f14e/w3af/plugins/output/text_file.py#L297
https://github.com/andresriancho/w3af/blob/356b14b975039706f4fd7f4f5db5b114cd75f14e/w3af/plugins/output/xml_file.py#L153

Reply:
Yes please